.chec.json

```
@@ -0,0 +1,10 @@
+{
+    "npm": "yarn",
+    "buildScripts": ["seed", "start"],
+    "dotenv": {
+        "NODE_ENV": "development",
+        "CHEC_PUBLIC_KEY": "%chec_public_key%",
+        "CHEC_API_URL": "%chec_api_url%",
+        "CHEC_SECRET_KEY": "%chec_secret_key%"
```

Not sure if what I'm suggesting here `%chec_skey%, %chec_pkey%`  are alias that also work. Did using `public` and `secret` work?
```suggestion
        "NODE_ENV": "development",
        "CHEC_PUBLIC_KEY": "%chec_pkey%",
        "CHEC_API_URL": "%chec_api_url%",
        "CHEC_SECRET_KEY": "%chec_skey%"
```

---

It's working the way it is :)